src/models/authentication.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String, Boolean
+
+from data.database import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    tenant_id = Column(Integer, index=True) # Not sure about this one
```

Comment:
Humm.. Name?

Reply:
And username

Reply:
ok, bear in mind that there are some providers  (oAuth) that they don't have any name, just the email is OK as it is unique and you don't need additional unique constrains on the DB. That being said, if you want we can do it optional and if not provided use the email as user-name